electrum/commands.py

```diff
@@ -1989,7 +1989,7 @@ async def get_submarine_swap_providers(self, query_time=15, wallet: Abstract_Wal
                 "max_forward_sat": offer.pairs.max_forward,
                 "max_reverse_sat": offer.pairs.max_reverse,
                 "min_amount_sat": offer.pairs.min_amount,
-                "provider_mining_fee": offer.pairs.mining_fee,
+                "prepayment": 2 * offer.pairs.mining_fee,
```

Comment:
also need to update this in the unit tests

Reply:
right